src/utils/StripedLoader.js

```diff
@@ -4,7 +4,7 @@ import { loading } from '../styles/generic'
 
 let StripedLoader = (Component, style = {}) =>
   observer(({ isLoading, ...props }) => (
-    <div style={{ height: '100%', ...style, ...(isLoading && loading) }}>
+    <div style={{ ...style, ...(isLoading && loading) }} className="loader">
```

Comment:
What's the classname for?

Reply:
I added it to identify the loaders better when inspecting, and figured it was potentially useful enough to leave in there. It's not actually *doing* anything though, so it's fine to take out if you want to.

Reply:
Yeah let's remove anything unused.